### PR TITLE
Support enabling KMS encryption for existing clusters

### PR DESCRIPTION
**Pull request by cPu1**
### Description

Adds support for enabling KMS encryption for existing clusters. 

### Checklist
- [x] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [x] Manually tested
- [x] Made sure the title of the PR is a good description that can go into the release notes
- [x] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

**Review by aclevername — APPROVED** [review]
LGTM

File: pkg/eks/update.go [hunk 1]
```diff
@@ -152,10 +154,131 @@ func (c *ClusterProvider) UpdatePublicAccessCIDRs(clusterConfig *api.ClusterConf
 	return c.waitForUpdateToSucceed(clusterConfig.Metadata.Name, output.Update)
 }
 
+// EnableKMSEncryption enables KMS encryption for the specified cluster
+func (c *ClusterProvider) EnableKMSEncryption(ctx context.Context, clusterConfig *api.ClusterConfig) error {
```
Comment (aclevername) [inline pkg/eks/update.go:158]: the provider grows :astonished: :cry:

**Review by michaelbeaumont — CHANGES_REQUESTED** [review]
Release notes

**Review by michaelbeaumont — APPROVED** [review]

**Comment by cPu1** [comment]
> Release notes

Updated the release notes.

**Pull request merged**